wetr.F90

```diff
@@ -197,44 +202,60 @@ subroutine getwetr(carma, igroup, rh, rdry, rwet, rhopdry, rhopwet, rc, h2o_mass
         end select
         
         rwet  = ((c1 * rdry**c2 / (c3 * rdry**c4 - log10(humidity))) + rdry**3)**(1._f / 3._f)
-      
+
         ! Determine the wet density from the wet radius.
         r_ratio  = (rdry / rwet)**3
         rhopwet = r_ratio * rhopdry + (1._f - r_ratio) * RHO_W
-      end if
-    end if
-    
+
+      end if ! irhswell(igroup) == I_GERBER
+
+      ! Mixed aerosol paremeterization (Pengfei Yu et al., JAMES, 2015) based on
+      ! Petters and Kreidenweis (ACP, 2007) hygroscopicity parameter kappa
+      if (irhswell(igroup) == I_PETTERS) then 
+	if (temp .le. 190._f) then
+	    rwet = rdry
+	    rhopwet = rhopdry
+	else ! temp > 190
+	    rwet = rdry * (1._f + humidity*kappa/(1._f-humidity))**(1._f/3._f)
+	    r_ratio  = (rdry / rwet)**3
```

Comment:
Have you plotted what this looks like at the 190 K boundary? Does it transition smoothly? Would it be better to use the value for 190 for everything less that 190 rather than returning the dry values?

Reply:
This is a valid concern. I have not plotted this, just copied it from Pengfei. It seems like it should be discontinuous, which is a problem. Your suggested fix is not straightforward, however, because temperature affects this indirectly via humidity and kappa, which are calculated elsewhere in the code.

Reply:
Looks like temperature and water are provided as optional parameters, so it seems like you could recalculate humidity (or crudely scale it by the some exponetial of the temperature ratio or something like that. Not sure about kappa. It is mostly the sum of the composition. What is the temperature dependence on kappa?

Reply:
@drmikemills kappa check

Reply:
Code changed so that when colder than 190K use relative humidity of 190K